client/components/SuggestionsList.tsx

```diff
@@ -1,15 +1,26 @@
-import React from 'react'
+import React from 'react';
 
-const SuggestionsList = ({ suggestions }) => {
-  // Group suggestions by their 'group' value
-  const groupedSuggestions = suggestions.reduce((groups, suggestion) => {
-    const group = suggestion.group || 'Uncategorized'
-    if (!groups[group]) {
-      groups[group] = []
-    }
-    groups[group].push(suggestion.phrase)
-    return groups
-  }, {})
+interface Suggestion {
+  phrase: string;
+  group: string;
+}
+
+interface SuggestionsListProps {
+  suggestions: Suggestion[];
+}
+
+const SuggestionsList: React.FC<SuggestionsListProps> = ({ suggestions }) => {
```

Comment:
Any chance we can amend this to https://stackoverflow.com/questions/59988667/typescript-react-fcprops-confusion

See discussion from TS team:

https://github.com/facebook/create-react-app/pull/8177

Reply:
On it boss